Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent any
+ 
+    stages {
+        stage('build') {
+            steps {
+                echo 'Building..'
+                withCredentials([file(credentialsId: 'env', variable: 'mySecretEnvFile')]){
+                    sh 'cp $mySecretEnvFile $WORKSPACE'
```

Comment:
personal question: so `$mySecretEnvFile` is a variable in your RPI that stores the location of an env that can be used when creating the build?

Reply:
another possibly silly question: when jenkins builds does it build from scratch every time? if it doesn't will there be a problem with copying the `$mySecretEnvFile` into `$WORKSPACE` every time we build?

Reply:
The file is stored in jenkins as a secret
if there are any changes to .env, we need to update in jenkins 
all build will use the up to date .env stored in jenkins